korrekt/src/circuit_analyzer/layouter.rs

```diff
@@ -83,7 +83,6 @@ impl<'a, F: Field> Layouter<F> for &'a mut AnalyticLayouter<F> {
 
         self.eq_table.insert(left, right);
         Ok(())
-        //todo!("handle instance columns") ticket created: https://quantstamp.atlassian.net/browse/ZKR-1238
```

Comment:
This appears to be a real `todo`; I will push a commit to check our internal ticket and re-add the todo.